tests/inc/test-update.php

```diff
@@ -318,6 +318,11 @@ function test_largo_transition_nav_menus() {
 	}
 
 	function test_largo_update_prominence_term_description_single() {
+		$term9 = get_term_by('slug', 'term-9', 'prominence', 'ARRAY_A');
+		if ( ! $term9 ) {
+			wp_insert_term( 'term-9', 'prominence', array( 'description' => 'Term Description 9' ) );
```

Comment:
We shouldn't have to create this term; it should have been created by this class' `setUp` method: https://github.com/rclations/Largo/blob/a773c5a2cfdbb7706c41085e8388f0d83b583e8e/tests/inc/test-update.php#L29-L31